webpack/stripes-translations-plugin.js

```diff
@@ -44,27 +51,30 @@ module.exports = class StripesTranslationPlugin {
       new webpack.ContextReplacementPlugin(/moment[/\\]locale/, filterRegex).apply(compiler);
     }
 
-    // Hook into stripesConfigPlugin to supply paths to translation files
-    // and gather additional modules from stripes.stripesDeps
-    StripesConfigPlugin.getPluginHooks(compiler).beforeWrite.tap({ name: 'StripesTranslationsPlugin', context: true }, (context, config) => {
-      // Add stripesDeps
-      for (const [key, value] of Object.entries(context.stripesDeps)) {
-        // TODO: merge translations from all versions of stripesDeps
-        this.modules[key] = value[value.length - 1];
+    if (this.federate) {
```

Comment:
It would be helpful to have some high-level comments at the top of this conditional that explains at a high-level how things are different for federated/monolithic builds. A bunch of the work is similar, but there is so much to do that the implementations end up far apart and are hard to compare.